blackbox/docs/sql/administration/ingestion_rules.txt

```diff
@@ -0,0 +1,98 @@
+.. highlight:: psql
+.. _administration-ingestion-rules:
+
+===============
+Ingestion Rules
+===============
+
+Introduction
+============
+
+In CrateDB ingestion rules are meant to be used together with an ingestion
+implementation, for the purpose of filtering data streams and storing them in
+the corresponding target tables.
```

Comment:
can you elaborate on what use case. maybe this document should be called "Ingestion Framework" and live somewhere else - ingestion rules are just an aspect of it.